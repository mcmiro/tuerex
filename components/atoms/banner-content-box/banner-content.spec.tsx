/* eslint-disable no-use-before-define */
import * as React from "react";
import {screen, render} from "@testing-library/react";
import BannerContent from "./";

describe("Test Banner Content component", () => {
	it("Should pass when name, variant, size and placeholder is additional", () => {
		render(
			<BannerContent
				link="/test"
				secondLink="/test2"
				title="test"
				content="test"
				buttonName="test"
				additionalButtonName="test"
				data-testid="banner-test"
			/>
		);

		const renderBannerContentComponent = screen.queryByTestId("banner-test");
		expect(renderBannerContentComponent).toBeDefined();
		expect(renderBannerContentComponent).toMatchSnapshot();
	});
});
