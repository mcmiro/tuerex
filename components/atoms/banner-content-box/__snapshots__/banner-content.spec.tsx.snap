// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Banner Content component Should pass when name, variant, size and placeholder is additional 1`] = `null`;
